skills/spry-debugging/references/route-debugging.md

```
@@ -0,0 +1,81 @@
+final middleware = [
+  // Scoped middleware first (specific → general)
+  MiddlewareRoute(path: '/admin', handler: adminMiddleware),
+  MiddlewareRoute(path: '/', handler: rootMiddleware),
+  // Global middleware last
+  MiddlewareRoute(path: '/**', handler: logger),
+];
+```
+
+### Verify with MCP
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)